app/build.gradle

```diff
@@ -89,4 +107,23 @@ android {
     lintOptions {
         abortOnError false
     }
+}
+
+// about.html is templatized so that we can automatically insert
+// our version string in to it at build time.
+task filterAbout {
+    inputs.files files(['src/main/templates/about.html',
+                        'src/main/AndroidManifest.xml'])
+    outputs.file 'src/main/assests/about.html'
+} << {
```

Comment:
No, that '<<' is gradle notation. The stuff before the << happens at the time the task is being configured. The stuff after the << happens during the actual build. It's the stuff you want to happen if the task needs to be completed.